src/components/highlighted-species-list/highlighted-species-list.js

```diff
@@ -2,8 +2,8 @@ import React, { useEffect, useState } from 'react';
 
 import { useLocale } from '@transifex/react';
 
-import EsriFeatureService from 'services/esri-feature-service';
-import MolService from 'services/mol';
+import EsriFeatureService from 'services/esri-feature/esri-feature-service';
+import MolService from 'services/mol/mol';
```

Comment:
Yes, maybe we can have the types on the definition folder this will avoid creating more folders for each service